ios_app_clip/lib/main.dart

```diff
@@ -13,13 +13,13 @@ void main() {
 // The same content is shown for both the main app target and in the App
 // Clip.
 class Demo extends StatefulWidget {
-  const Demo({Key key}) : super(key: key);
+  const Demo({Key? key}) : super(key: key);
 
   @override
-  State<StatefulWidget> createState() => _DemoState();
+  State<Demo> createState() => DemoState();
```

Comment:
`State<Demo> createState() => _DemoState();`

Reply:
Which change are you looking for here - the return of the leading underscore? If I'm reading this right, that's now frowned upon by the updated set of lints.

Reply:
The way I've been writing the code is keeping the `State` object private, with it's leading underscore, and make the return type of the `createState()` method be `State<WidgetName>`. You can see an example of this [on DartPad](https://dartpad.dev/?id=85e77d36533b16647bf9b6eb8c03296d&channel=beta) complying with Flutter beta's linter. Specifically

```dart
class VariousDiscs extends StatefulWidget {
  final int numberOfDiscs;

  const VariousDiscs(this.numberOfDiscs);

  @override
  State<VariousDiscs> createState() => _VariousDiscsState();
}

class _VariousDiscsState extends State<VariousDiscs> {
  // redacted
}
```

Reply:
Updated